### ISSUE TITLE
How to reproduce the results of the self attention baseline in the paper "Pay Less Attention with Lightweight and Dynamic Convolutions (Wu et al., 2019)"

### ISSUE THREAD

**Issue by zhaoguangxiang** [pytorch/fairseq#534]
e.g. 39.2 rouge scores  on the cnndm summarzation dataset, 24 bleu scores on the wmt-17 zh-en translation dataset.

**Comment by huihuifan** [comment]
For summarization, we decoded with a min/max length and trigram blocking with beam search. We trimmed  input articles on anonymized cnn dailymail 400 tokens and used bpe 30k, transformer base architecture. This is described in our paper. 

For zh-en, we have pretrained models available for our lightweight and dynamic convolutions (https://github.com/pytorch/fairseq/tree/master/examples/pay_less_attention_paper). For the self-attention baseline, we follow the Transformer Big architecture and preprocess the zh-en dataset as described in the paper. We also detail the dropout we used in the paper - 0.25.

**Comment by zhaoguangxiang** [comment]
> For summarization, we decoded with a min/max length and trigram blocking with beam search. We trimmed  input articles on anonymized cnn dailymail 400 tokens and used bpe 30k, transformer base architecture. This is described in our paper.
> For zh-en, we have pretrained models available for our lightweight and dynamic convolutions (https://github.com/pytorch/fairseq/tree/master/examples/pay_less_attention_paper). For the self-attention baseline, we follow the Transformer Big architecture and preprocess the zh-en dataset as described in the paper. We also detail the dropout we used in the paper - 0.25.

Thank you! Will you release the scripts to reproduce the above results in fairseq?

**Comment by GeorgeS2019** [comment]
@myleott so far there has been many discussions on the data preparation scripts on how to create the binary data needed to reproduce the provided binary newstest2017 data for used in the [pre-trained zh-en model](https://github.com/pytorch/fairseq/tree/master/examples/pay_less_attention_paper). 

It would be great if the script is provided so more users can use fairseq to learn zh to en translation.
